l10n_de_tax_statement/models/l10n_de_tax_statement_2019.py

```diff
@@ -282,39 +260,37 @@ def _finalize_lines_2019(lines):
     _35b = lines['35']['tax']
     lines['36']['tax'] = lines['36']['tax'] * -1
     _36b = lines['36']['tax']
-    # calculate reverse of lines 48 - line 52 base
+    # calculate reverse of lines 48 - line 50 base
     lines['48']['base'] = lines['48']['base'] * -1
     lines['49']['base'] = lines['49']['base'] * -1
     lines['50']['base'] = lines['50']['base'] * -1
-    lines['51']['base'] = lines['51']['base'] * -1
-    lines['52']['base'] = lines['52']['base'] * -1
-    # calculate reverse of line 55 - line 61
+    # calculate reverse of line 53 - line 59
+    lines['53']['tax'] = lines['53']['tax'] * -1
+    _53b = lines['53']['tax']
+    lines['54']['tax'] = lines['54']['tax'] * -1
+    _54b = lines['54']['tax']
     lines['55']['tax'] = lines['55']['tax'] * -1
     _55b = lines['55']['tax']
     lines['56']['tax'] = lines['56']['tax'] * -1
     _56b = lines['56']['tax']
     lines['57']['tax'] = lines['57']['tax'] * -1
-    _57b = lines['57']['tax']
+    _57b = lines['59']['tax']
     lines['58']['tax'] = lines['58']['tax'] * -1
-    _58b = lines['58']['tax']
+    _58b = lines['60']['tax']
     lines['59']['tax'] = lines['59']['tax'] * -1
     _59b = lines['59']['tax']
-    lines['60']['tax'] = lines['60']['tax'] * -1
-    _60b = lines['60']['tax']
-    lines['61']['tax'] = lines['61']['tax'] * -1
-    _61b = lines['61']['tax']
-    # calculate line 53
-    lines['53']['tax'] = lines['53']['tax'] * 1
-    _53b = lines['53']['tax'] + _26b + _27b + _28b + _30b + _33b + _34b \
-           + _35b + _36b + _48b + _49b + _50b + _51b + _52b
-    # calculate line 62
-    lines['62']['tax'] = lines['62']['tax'] * 1
-    _62b = lines['62']['tax'] + _53b - _55b - _56b - _57b \
-           - _58b - _59b - _60b - _61b
-    # calculate line 66
-    lines['66']['tax'] = lines['66']['tax'] * 1
-    _66b = lines['66']['tax'] + _62b + _64b + _65b
-    # update lines 53, 62, 66
-    lines['66'].update({'tax': _66b})
-    lines['53'].update({'tax': _53b})
-    lines['62'].update({'tax': _62b})
+    # calculate line 51
+    lines['51']['tax'] = lines['51']['tax'] * 1
+    _51b = lines['51']['tax'] + _26b + _27b + _28b + _30b + _33b + _34b \
+           + _35b + _36b + _48b + _49b + _50b
+    # calculate line 60
+    lines['60']['tax'] = lines['60']['tax'] * 1
+    _60b = lines['60']['tax'] + _51b - _53b - _54b - _55b \
+           - _56b - _57b - _58b - _59b
+    # calculate line 64
+    lines['64']['tax'] = lines['64']['tax'] * 1
+    _64b = lines['64']['tax'] + _60b + _62b + _63b
```

Comment:
@tv-openbig  here _62b and _63b are not defined, it seems the definition was deleted. Could you check?

Reply:
Correct, in next commit i will change that.